src/components/BareMetalHosts/StatusComponents.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Popover } from '@patternfly/react-core';
+
+import { getHostStatusText } from '../../selectors/host/selectors';
+
+export const IconAndText = ({ icon, text }) => {
+  const className = `pficon ${icon} co-icon-and-text__icon`;
+  return (
+    <span className="co-icon-and-text">
+      <span aria-hidden="true" className={className} />
```

Comment:
Icon component is available from patternfly
`<Icon type="pf" name="error-circle-o" />`

Reply:
Done